sceptre_cdk_handler/cdk_builder.py

```diff
@@ -1,8 +1,11 @@
+import json
 import logging
```

Comment:
I needed to refactor the classes in this module since there was a lot of things in common between the CdkBuilder classes but also each type of CdkBuilder needed its own way of doing certain things. The changes in this module are largely just moving code around and then adding the CdkJsonBuilder.